.github/workflows/ci.yml

```
@@ -81,6 +81,14 @@ jobs:
       - name: Run cargo fmt
         run: cargo fmt --all --check
 
+      - name: Install Taplo
+        uses: uncenter/setup-taplo@v1
+        with:
+          version: "0.9.3"
+
+      - name: run `taplo fmt`
```

```suggestion
      - name: Run taplo fmt
```
would be consistent with the other projects.